### PR TITLE
Crud requests

### PR DESCRIPTION
**Pull request by alexamsv**

### CODE REVIEW

File: Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @Mapping(target = "active", source = "request.active")
+    Request requestRequestToRequest(RequestRequest request, User customer, Service service);
+    @Mapping(target = "customerId", source = "request.customer.id")
+    @Mapping(target = "serviceId", source = "request.service.id")
+    RequestResponse requestToRequestResponse(Request request);
+
+    List<RequestResponse> toRequestResponseList(List<Request> requests);
+
+    void requestFromRequest(RequestRequest requestRequest, @MappingTarget Request request);
+
```
Comment (DameStoilovski99) [inline Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java:34]: delete empty rows

File: Backend/src/main/java/com/service/marketplace/dto/request/RequestRequest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class RequestRequest {
+
```
Comment (DameStoilovski99) [inline Backend/src/main/java/com/service/marketplace/dto/request/RequestRequest.java:11]: Rename this class to "RequestToCreateDto"

Comment (alexamsv) [reply Backend/src/main/java/com/service/marketplace/dto/request/RequestRequest.java:11]: > Rename this class to "RequestToCreateDto"

okay

File: Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java (hunk 1 above)
Comment (alexamsv) [reply Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java:34]: > delete empty rows

we use toRequestResponseList in the getAllRequests() method in RequestServiceImpl and requestFromRequest in updateRequest.

Comment (DameStoilovski99) [reply Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java:29]: The empty rows below the methods

Comment (alexamsv) [reply Backend/src/main/java/com/service/marketplace/mapper/RequestMapper.java]: > The empty rows below the methods

okay

**Review by DameStoilovski99 — APPROVED** [review]

**Pull request merged**